src/esp/bindings/MetadataMediatorBindings.cpp

```diff
@@ -25,6 +25,15 @@ void initMetadataMediatorBindings(py::module& m) {
           R"(The currently active dataset being used.  Will attempt to load
             configuration files specified if does not already exist.)")
 
+      /* --- Methods --- */
+      .def(
+          "dataset_exists", &MetadataMediator::sceneDatasetExists,
+          R"(Checks whether the passed name references an existing scene dataset)",
```

Comment:
and returns a bool or throws an exception?

Reply:
Returns boolean.

Reply:
I would say returns whether it exists instead of check since check ambiguous.